string essid
float64 frequency
float64 gain
float64 power
geometry_msgs/Pose pose
